datamade/la-metro-councilmatic
# Add event calendar link to homepage and events page

## Overview

See title.

### Checklist


### Demo

<img width="1552" alt="Screen Shot 2022-05-25 at 1 43 12 PM" src="https://user-images.githubusercontent.com/12176173/170344813-ad7662d4-4324-4a17-8bb7-75a1a39ba3d2.png">

<img width="1552" alt="Screen Shot 2022-05-25 at 1 43 02 PM" src="https://user-images.githubusercontent.com/12176173/170344808-445e7a7b-3ced-45ca-a503-3e3e835939d9.png">

## Testing Instructions

 * Pull down this app and run it.
 * View the links on the homepage and confirm they look good and lead to the correct document.
 * View the events listing and confirm the link looks good and leads to the correct document.

Handles #853
